.idea/misc.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
revert this file changes

Reply:
@yash10019coder  the misc.xml have been added by default so I have deleted the file after committing. Is it necessary to add that back?

Reply:
revert back

Reply:
done